aws/ci_build

```diff
@@ -226,11 +225,15 @@ def mark_dtt_green
 end
 
 # Log details of a build to metrics database.
-# @param [Symbol] environment - :success or :failed
+# TODO: (suresh) This method will eventually be used to write build status for all managed environments.
+# @param [Symbol] environment
 # @param [String] commit_hash
-# @param [String] status
+# @param [String] status - :start, :success, :failed
```

Comment:
nit: the type is listed as `String`, but the values you listed are `Symbol`s. Probably best to list the type as `Symbol` if you care which is used, or `Symbol|String` otherwise.